cluster-autoscaler/integration/framework.go

```diff
@@ -206,3 +244,121 @@ func (driver *Driver) scaleWorkload(workloadName string, replicas int32) error {
 	}
 	return nil
 }
+
+func (driver *Driver) getFirstNode() (*v1.Node, error) {
```

Comment:
make a single function called `getOldestAndLatestNode` to get both the nodes, instead of two different functions

Reply:
There is no test where both the nodes are required simultaneously, so what’s the benefit of this change?

Reply:
Done.